src/desktop/src/libs/realm.js

```diff
@@ -0,0 +1,27 @@
+/* global Electron */
+import base64 from 'base64-js';
+import { randomBytes } from './crypto';
+
+const ALIAS_REALM = 'realm_enc_key';
+/**
+ * Gets encryption key for realm.
+ * - Checks keychain if there is already an encryption key stored for realm data.
+ * - Returns encryption key stored in keychain (if exists).
+ * - Otherwise generates a new one and stores it in keychain.
+ *
+ * @method getEncryptionKey
+ * @returns {Promise}
+ */
+export const getEncryptionKey = () => {
+    return Electron.readKeychain(ALIAS_REALM).then((encryptionKey) => {
+        if (encryptionKey === null) {
+            return Promise.resolve(randomBytes(64)).then((bytes) =>
+                Electron.setKeychain(ALIAS_REALM, base64.fromByteArray(encryptionKey)).then(
+                    () => new Uint8Array(bytes),
+                ),
+            );
+        }
+
+        return base64.toByteArray(encryptionKey);
```

Comment:
There is no need to add an additional dependency for he string transformations. An `Uint8Array`  can be transformed to string via `encryptionKey.toString()` and back to Uint8Array as  `Uint8Array.from(encryptionKey.split(","))`

Reply:
@rihardsgravis Addressed in https://github.com/iotaledger/trinity-wallet/pull/1018/commits/039a2fa4560fb963e1ae21101f62c615ad0e93bc